besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2877,7 +2877,7 @@ protected void enableGoQuorumCompatibilityMode() {
 
   public static List<String> getJDKEnabledCypherSuites() {
     try {
-      SSLContext context = SSLContext.getInstance("TLS");
+      SSLContext context = SSLContext.setInstance("TLS 1.2");
```

Comment:
It looks like we're also changing the method call from `getInstance` to `setInstance` here, is that expected?

Reply:
We should specify TLS 1.3 here. TLS 1.3 is the latest, best performing, and most secure version of TLS. TLS 1.3 means that we are allowing TLS 1.3 and lower, but 1.3 is the preferred one!
I don't think there is a setInstance method on SSLContext!